d3b_dff_cli/modules/validation/check_manifest.py

```diff
@@ -43,14 +45,30 @@ def validate_row(row, rules):
                 if col == "file_size" and row.get("file_format", "").lower() in ["fastq", "bam", "cram"]:
                     greater_than_value = consequence.get("greater_than")
                     if greater_than_value:
-                        try:
-                            file_size_in_gb = float(row.get(col, 0)) / (1024 * 1024 * 1024)  # Convert to GB
-                            if file_size_in_gb <= float(greater_than_value.rstrip(" GB")):
-                                error_messages.append(f"Warning: *{col}* less than {greater_than_value}")
-                        except ValueError:
-                            error_messages.append(f"*{col}* is not a valid numeric value")
+                        experiment = row.get("experiment_strategy", "").lower()
+                        if experiment in ["wgs", "wxs", "wes"]:
+                            greater_than_value = "1 GB"
+                            minum_value = 1_000_000_000 # WGS/WXS should be greater than 1G
+                        else:
+                            greater_than_value = consequence.get("greater_than")
+                            minum_value = float(greater_than_value.rstrip("M"))*1000_000 # Other experimental strategy should be greater than the specified value.
+                        
+                        if pd.notna(cell_value):
+                            fize_size_str = str(cell_value).lower()
 
+                            # support file size formats in bytes, megabytes, and gigabytes: 100, 100M, 100MB, 10G, 10GB
+                            try:
+                                if 'm' in fize_size_str:
+                                    size_byte = float(re.sub('[^0-9.]', '', fize_size_str)) * 1_000_000
```

Comment:
Are we planning to update documentation to ask users to provide us the unit of measurement they are submitting for file size? If so, I would advocate for a separate column for file size unit vs. embedding in the file size column.

Reply:
> Are we planning to update documentation to ask users to provide us the unit of measurement they are submitting for file size? If so, I would advocate for a separate column for file size unit vs. embedding in the file size column.

I’ve updated the doc in the [shinyapp](https://d3b-rstudio-connect-public-prd.d3b.io/manifest-template/). I think this is an easier way than adding a separate column, as I don't want to include too many columns in the template manifest, especially since we’re planning to include more fields as more data types are incorporated.
<img width="1222" alt="image" src="https://github.com/user-attachments/assets/cdfa3380-b9b8-46d1-b664-18f6bb4944ef">

Reply:
As discussed in the channel, it should only be bytes still.